src/kubernetes-ingress-controller/guides/using-kongplugin-resource.md

```diff
@@ -436,6 +470,10 @@ credentials:
 kongconsumer.configuration.konghq.com/harry configured
 ```
 
+**Please note:** validation of the configuration fields is left to the user
+by default. It is advised to setup and use the admission validating controller
+to catch user errors.
+
```

Comment:
This should be earlier, in the introduction, use a [warning admonition](https://docs.konghq.com/contributing/markdown-rules/#admonitions), and include a link to the admission controller guide (`/deployment/admission-webhook/`).

The guide doesn't strictly require this (and the guides are generally simplified for brevity), so this should call out that this is mostly a production concern, e.g.

> The controller does not validate configuration by default. Although not required for this guide, production instances should [enable the admission controller](/deployment/admission-webhook/) to validate plugins.

Reply:
78d3c58